honeybee_schema/radiance/modifierset.py

```diff
@@ -0,0 +1,173 @@
+"""ModifierSet Schema"""
+from pydantic import Field, constr
+from typing import List, Union
+
+from .._base import IDdBaseModel
+from .modifier import ModifierBase, Void
+
+
+class BaseModifierSet(IDdBaseModel):
```

Comment:
This should be inheriting from [`NoExtraBaseModel`](https://github.com/ladybug-tools/honeybee-schema/blob/master/honeybee_schema/_base.py#L5) since these sub-sets do not have any `identifier` or `display_name` keys.  Only the `ModifierSet` has these two keys.

Reply:
I would really recommend looking at how the ConstructionSet is described in Pydantic and this is a good example to follow:
https://github.com/ladybug-tools/honeybee-schema/blob/master/honeybee_schema/energy/constructionset.py